packages/swapper/src/swappers/zrx/utils/constants.ts

```diff
@@ -11,3 +11,5 @@ export const MAX_GAS_PRICE = '1000000000000' // 1000 GWei
 export const MAX_ZRX_TRADE = '100000000000000000000000000'
 export const DEFAULT_SOURCE = [{ name: '0x', proportion: '1' }]
 export const DEFAULT_SLIPPAGE = '3.0' // 3%
+export const DEFAULT_ETH_PATH = `m/44'/60'/0'/0/0`
```

Comment:
can we add a todo above this to remove once with add and make this optional in chain adapters?